internal/verifier/change_stream.go

```diff
@@ -15,12 +15,15 @@ import (
 	"golang.org/x/exp/constraints"
 )
 
+const fauxDocSizeForDeleteEvents = 1024
+
 // ParsedEvent contains the fields of an event that we have parsed from 'bson.Raw'.
 type ParsedEvent struct {
 	ID          interface{}          `bson:"_id"`
 	OpType      string               `bson:"operationType"`
 	Ns          *Namespace           `bson:"ns,omitempty"`
 	DocKey      DocKey               `bson:"documentKey,omitempty"`
+	DocSize     *int                 `bson:"fullDocument,omitempty"`
```

Comment:
The BSON tag should probably be `documentSize`.

Reply:
oops! Good catch.